plugins/git/lazygit.nix

```diff
@@ -0,0 +1,93 @@
+{
+  config,
+  lib,
+  helpers,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.vim-plugin.mkVimPlugin config {
+    name = "lazygit";
+    originalName = "lazygit.nvim";
+    defaultPackage = pkgs.vimPlugins.lazygit-nvim;
+    globalPrefix = "lazygit_";
+
+    extraPackages = [pkgs.lazygit pkgs.git];
+
+    maintainers = [helpers.maintainers.AndresBermeoMarinelli];
+
+    settingsOptions = {
+      floating_window_winblend = helpers.defaultNullOpts.mkNullable (types.ints.between 0 100) "0" ''
+        Set the transparency of the floating window.
+      '';
+
+      floating_window_scaling_factor =
+        helpers.defaultNullOpts.mkNullable types.numbers.nonnegative
+        "0.9" "Set the scaling factor for floating window.";
```

Comment:
```suggestion
      floating_window_scaling_factor = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) "0.9" ''
        Set the scaling factor for floating window.
      '';
```

Reply:
Maybe I'm wrong on this one. Can it be > 1 ?

Reply:
I'm not sure. The source code only uses it as a multiplicative factor: look [here](https://github.com/kdheepak/lazygit.nvim/blob/0ada6c6e7e138df92f5009b6952f4ac41248305a/lua/lazygit/window.lua#L19)

So I thought there were no limitations as long as it was non negative.

Reply:
Ok, let's keep it this way then.